my_list.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'module'
```

Comment:
- [ ] Running your code through this file or IRB through this error:
![image](https://user-images.githubusercontent.com/33767951/149831151-730caf31-b27d-4dba-83f4-06de2824ca06.png)
 Please import the `MyEnumerable` module correctly.
- [ ] Please, add instructions in your README file on how to get started with this project.

Reply:
Thanks for the feedback. I have fixed the issue.